beakerx/beakerx/install.py

```diff
@@ -80,36 +74,44 @@ def _install_css():
     shutil.copyfile(os.path.join(src_base, 'custom.css'), os.path.join(dst_base, 'custom.css'))
 
 
-def _install_kernels():
+def install_kernel(kernel_name):
+    print('---------------------------------')
     base_classpath = _classpath_for('base')
-
-    for kernel in _all_kernels():
-        kernel_classpath = _classpath_for(kernel)
-        classpath = json.dumps(os.pathsep.join([base_classpath, kernel_classpath]))
-        template = pkg_resources.resource_string(
-            'beakerx', os.path.join('kernel', kernel, 'kernel.json'))
-        contents = Template(template.decode()).substitute(PATH=classpath)
-
-        with tempfile.TemporaryDirectory() as tmpdir:
-            with open(os.path.join(tmpdir, 'kernel.json'), 'w') as f:
-                f.write(contents)
-            install_cmd = [
-                'jupyter', 'kernelspec', 'install',
-                '--sys-prefix', '--replace',
-                '--name', kernel, tmpdir
-            ]
-            subprocess.check_call(install_cmd)
-
-def _uninstall_kernels():
-    for kernel in _all_kernels():
-        uninstall_cmd = [
-            'jupyter', 'kernelspec', 'remove', kernel, '-y', '-f'
+    kernel_classpath = _classpath_for(kernel_name)
+
+    print('base_classpath: ', base_classpath)
+    print('kernel_classpath: ', kernel_classpath)
+
+    print("test: ", _classpath_for("java"))
+
+    print('---------------------------------')
+    classpath = json.dumps(os.pathsep.join([base_classpath, kernel_classpath]))
+    template = pkg_resources.resource_string(
+        'beakerx_' + kernel_name, 'kernel.json')
+    contents = Template(template.decode()).substitute(PATH=classpath)
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        with open(os.path.join(tmpdir, 'kernel.json'), 'w') as f:
+            f.write(contents)
+        install_cmd = [
+            'jupyter', 'kernelspec', 'install',
+            '--sys-prefix', '--replace',
+            '--name', kernel_name, tmpdir
         ]
-        subprocess.check_call(uninstall_cmd)
+        subprocess.check_call(install_cmd)
+
+
+def uninstall_kernel(kernel_name):
+    uninstall_cmd = [
+        'jupyter', 'kernelspec', 'remove', kernel_name, '-y', '-f'
```

Comment:
What would happen here if I already have a `jupyter` in my path? Is care taken somewhere else to make sure the right one is called?

Reply:
No, It use jupyter from PATH. It is made the same as install kernel and nbextension